charts/data-dashboard-backend/templates/deployment.yaml

```diff
@@ -58,14 +58,14 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /api/health
+              path: {{ .Values.path }}/health
               port: http
               httpHeaders:
                 - name: Accept
                   value: application/json
           readinessProbe:
             httpGet:
-              path: /api/health
+              path: {{ .Values.path }}/health
```

Comment:
Not really related to this PR but some of our components are pointed to a sub domain and some other to prefixes in the main domain and it's not clear for me when we use which one. I think we should have an agreement on it. I'll make an issue for it.

Reply:
Yes that is true. It's good to have this configurable.